### DISCUSSION TITLE
Extension Launcher Question.

### DISCUSSION THREAD

**Discussion by PingzzYt** [3kh0/ext-remover#702] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
how would I go about filling in the extension icon box? I cant seem to figure it out, ( I'm trying to download Tunnelbear VPN). Also, has another way to run the program come out if bookmarks are disabled

Thanks in advance :)

**Comment by 3kh0** [comment, marked as answer]
i am confused by your question...
